pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -589,6 +593,21 @@ func (v *vpnShoot) getEnvVars() []corev1.EnvVar {
 				Value: v.values.ReversedVPN.Header,
 			},
 		)
+		if initContainer {
+			envVariables = append(envVariables,
+				corev1.EnvVar{
+					Name:  "EXIT_AFTER_CONFIGURING_KERNEL_SETTINGS",
+					Value: "true",
+				},
+			)
```

Comment:
I assume the init container should not. need any other env var other than `EXIT_AFTER_CONFIGURING_KERNEL_SETTINGS`.
Instead of append to the existing envVars, let's rather only return the `EXIT_AFTER_CONFIGURING_KERNEL_SETTINGS` env var for the init container.

Reply:
Adapted as requested with latest commit.